clevertap-core/src/main/java/com/clevertap/android/sdk/network/api/DefineTemplatesRequestBody.kt

```diff
@@ -23,15 +23,15 @@ private fun Collection<CustomTemplate>.toJSON(): JSONObject {
         putObject("definitions") {
             for (template in templates) {
                 putObject(template.name) {
-                    put("type", template.type.stringName)
+                    put("type", template.type.toString())
```

Comment:
hope this change is intentional.

Reply:
Yes, I've changed `toString()` to return `stringName`.